recipes/python-lambda-4dn/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "python-lambda-4dn" %}
+{% set version = "0.12.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "a86d8b438e041cc7b0d144e16f24bae16cdbb4003ef2b55bbc89b187f2da7e0f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

https://github.com/conda-forge/staged-recipes#7-when-or-why-do-i-need-to-use--python---m-pip-install---vv
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv""
```